### ISSUE TITLE
Thanks a lot!

### ISSUE THREAD

**Issue by oktaygit** [tobywf/tobywf.github.io#1]
Page: https://tobywf.com/2017/08/unicode-csv-excel/

Hey Toby, thank you very very much for sharing this fantastic tip! That's a really awesome one!

**Comment by markhu** [comment]
Yes, the `encoding='utf-8-sig'` works a charm. If only that was the default!
